DEFAULT_CONFIG.json5

```diff
@@ -25,6 +25,24 @@
     endpoints: [
       // "<proto>/<address>"
     ],
+    //  /// Global connect retry configuration, 
```

Comment:
I believe some of those parameters are applied as default parameters, therefore they should be uncommented. The idea of the `DEFAULT_CONFIG.json5` is that everything applied by default should be uncommented. I.e., starting zenoh with no configuration or with the default configuration should lead to the same behaviour.